<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Calcul d'alcoolémie</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelSexe">
         <property name="text">
          <string>Sexe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonSexeHomme">
         <property name="text">
          <string>Homme</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonSexeFemme">
         <property name="text">
          <string>Femme</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Poids</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="poidInput">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0" columnstretch="1,1,1">
       <item row="3" column="1">
        <widget class="QLabel" name="degreVin">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(52, 152, 219);</string>
         </property>
         <property name="text">
          <string>12°</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="quantiterClAperitif"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 128, 185);</string>
         </property>
         <property name="text">
          <string>Autres</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="degreAperitif">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(52, 152, 219);</string>
         </property>
         <property name="text">
          <string>35°</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="degreBiere">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(52, 152, 219);</string>
         </property>
         <property name="text">
          <string>5°</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="quantiterClBiere"/>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="quantiterClCustom"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 128, 185);</string>
         </property>
         <property name="text">
          <string>Bière</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="quantiterClVin"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 128, 185);</string>
         </property>
         <property name="text">
          <string>Apéritif</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 128, 185);</string>
         </property>
         <property name="text">
          <string>% en volume</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_16">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 128, 185);</string>
         </property>
         <property name="text">
          <string>Quantité en cl</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 128, 185);</string>
         </property>
         <property name="text">
          <string>Vin</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 128, 185);</string>
         </property>
         <property name="text">
          <string>Boissons</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(41, 128, 185);
gridline-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Boisson consommées</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="degreCustom"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelResult">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="buttonDeleteAll">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(231, 76, 60);</string>
         </property>
         <property name="text">
          <string>Effacer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="validerButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(46, 204, 113);</string>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
